tasks/st0236_set_cache_paths_to_default.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
 # shellcheck disable=SC2230
+# DEPRECATION:
+# This script is now Deprecated and will be removed in a further update
 declare PT__installdir
 source "$PT__installdir/bash_task_helper/files/task_helper.sh"
 
 if [ -n  "$(facter -p pe_build)" ]
 then
-	task-suceed "success - Not an agent node"
+  echo "This Script is now Deprecated and has been replaced by the task-output function, which provides the same functionality"
```

Comment:
The message doesn't seem necessary to me?  This task is leveraging the `task-suceed` and other functions provided by the bash helper, so we can just change how we use it here.  I think we can just remove all of these `echo`s about `task-output`.